src/features/home/layouts/HomeLayout.tsx

```diff
@@ -71,6 +71,13 @@ const HomeLayout: FC<Props> = ({ children, title }) => {
             sx={{ textTransform: 'none' }}
             value="feed"
           />
+          <Tab
+            component={NextLink}
+            href="/my/settings"
+            label="Settings"
```

Comment:
A lil' localisation for this queen too

Reply:
Fixed in d2b2cbabcace45b5b256e7c0a9e2d27a807eeee2.